dcicutils/obfuscation_utils.py

```diff
@@ -115,6 +119,14 @@ def has_values_to_obfuscate(dictionary: dict) -> bool:
     for key, value in dictionary.items():
         if isinstance(value, dict):
             dictionary[key] = obfuscate_dict(value, show=False, inplace=False, obfuscated=obfuscated)
+        elif isinstance(value, list):
```

Comment:
I guess you picked this up from my branch (unless you somehow made precisely the same change :) 
Question: This also does not handle passing in a list at top-level - do you think it should? Of course, the function name (obfuscate_dict) would be off, but would probably good to handle that too. If so I can take care of that (and new tests).